sys/mips/cheri/cheri_debug.c

```diff
@@ -51,8 +51,7 @@
 static inline void
 db_print_cap(const char* msg, void * __capability cap)
 {
-	db_printf("%s" _CHERI_PRINTF_CAP_FMT "\n", msg,
-	    _CHERI_PRINTF_CAP_ARG(cap));
+	db_printf("%s%#.16lp\n", msg, cap);
```

Comment:
Precision here is probably fine.  We can probably actually axe this function entirely and just inline it at this point, but that can be a separate followup for someone who really wants to do that.